plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -82,18 +73,13 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 		await expect(
 			page.getByText( 'Product draft updated. ' )
 		).toBeVisible();
-		await page.click( '.updated.notice .notice-dismiss' );
```

Comment:
No point dismissing the notice, doesn't serve a purpose for the test.

Reply:
Agree! Thanks for cleaning this up